three-tier-main.tf

```diff
@@ -286,7 +288,7 @@ resource "aws_lb_listener" "external-elb" {
 }
 
```

Comment:
**Tenable.cs remediation**
 
**[MEDIUM]: Ensure IAM authentication is enabled for AWS RDS instances**
 
**Vulnerability**
IAM authentication is disabled for AWS RDS instances which prevents centralized access.
 
**Remediation**
IAM authentication can be enabled for AWS RDS instances using AWS Console.
 In AWS Console - 
 1. Sign in to the AWS Console and go to the AWS RDS console.
 2. Select Databases in the navigation pane.
 3. Select the DB instance to modify.
 4. Select Modify.
 5. Select password and IAM database authentication, in the Database Authentication section, to enable IAM database authentication.
 6. Select Continue.
 7. Select immediately in the Scheduling of modifications section. to apply changes immediately.
 8. Select Modify DB instance.

source: tenablecs-